add readme minio-distributed.yaml minio-standalone.yaml (#208)
hwameistor/hwameistor: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt

HwameiStor-MinIO Cloud Native Solution

> 1. Solution Introduction

Reviewer: ```suggestion
## Solution Introduction
```

HwameiStor is a high availability local storage system for cloud-native stateful workloads. It creates a local storage resource pool for centrally managing all disks such as HDD, SSD, and NVMe. It uses the CSI architecture to provide distributed services with local volumes and provides data persistence capabilities for stateful cloud-native workloads or components.

> MinIO is a High Performance Object Storage released under GNU Affero General Public License v3.0. It is API compatible with Amazon S3 cloud storage service. Use MinIO to build high performance infrastructure for machine learning, analytics and application data workloads.

Reviewer: ```suggestion
MinIO is a High Performance Object Storage released under GNU Affero General Public License v3.0. It provides APIs compatible with Amazon S3 cloud storage service. You can use MinIO to build high performance infrastructure for machine learning, analytics and application data workloads.
```

> MinIO is used as the main storage for cloud-native applications. Compared with traditional object storage, cloud-native applications require higher throughput and lower latency. These are the performance indicators that MinIO can achieve, with read/write speeds of up to 183 GB/sec and 171 GB/sec.

Reviewer: ```suggestion
MinIO is used as the main storage for cloud-native applications. Compared with the traditional object storage, MinIO can provide higher throughput and lower latency required by cloud-native applications. For example, MinIO can achieve the read and write capacity of up to 183 GB and 171 GB per second.
```

> The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO. HwameiStor is a storage system that meets the requirements of the cloud-native era. It has the advantages of high performance, high availability, automation, low cost, and rapid deployment, and can replace expensive traditional SAN storage.

Reviewer: ```suggestion
The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage is a good choice with the competitive read and write performance, which can undoubtedly provides performance guarantee for MinIO. HwameiStor is such a local storage system that waves with the cloud-native era. High performance, high availability, automation, low cost, rapid deployment, and more benefits attract you to replace those expensive and traditional storage schemes such as SAN (Storage Area Network).
```

Reviewer 2: "Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO."

Does it mean Hwameistor's Local Storage?

> 2. Solution Validation

Reviewer: ```suggestion
## Solution Validation
```

k8s-10-6-163-51-sdf   k8s-10-6-163-51           Claimed
k8s-10-6-163-52-sda   k8s-10-6-163-52           Inuse
k8s-10-6-163-52-sdb   k8s-10-6-163-52           Unclaimed
k8s-10-6-163-53-sda   k8s-10-6-163-53           Inuse
k8s-10-6-163-53-sdb   k8s-10-6-163-53           Claimed
k8s-10-6-163-53-sdc   k8s-10-6-163-53           Claimed
k8s-10-6-163-53-sdd   k8s-10-6-163-53           Claimed
k8s-10-6-163-53-sde   k8s-10-6-163-53           Claimed
k8s-10-6-163-53-sdf   k8s-10-6-163-53           Claimed
k8s-10-6-163-54-sda   k8s-10-6-163-54           Inuse
k8s-10-6-163-54-sdb   k8s-10-6-163-54           Unclaimed
k8s-10-6-163-54-sdc   k8s-10-6-163-54           Unclaimed
k8s-10-6-163-54-sdd   k8s-10-6-163-54           Unclaimed
k8s-10-6-163-54-sde   k8s-10-6-163-54           Unclaimed
k8s-10-6-163-54-sdf   k8s-10-6-163-54           Unclaimed

View StorageClass status

[root@k8s-10-6-163-52 minio-test1]# kubectl get sc
NAME                         PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
hwameistor-storage-lvm-hdd   lvm.hwameistor.io   Delete          WaitForFirstConsumer   true                   45h

> 3. Deploy MinIO on Kubernetes

Reviewer: ```suggestion
## Deploy MinIO on Kubernetes
```

> 3.1 Standalone deployment

Reviewer: ```suggestion
### Standalone deployment
```

> Deploy based on official helm chart and install MinIO chart.

Reviewer 2: ```suggestion
Deploy based on the official helm chart and install the MinIO chart.
```

[root@k8s-10-6-163-52 minio-test1]# helm repo add minio https://helm.min.io

[root@k8s-10-6-163-52 minio-test1]# helm repo list | grep minio
minio         	https://helm.min.io/

> Standalone mode deployment.

Reviewer 2: ```suggestion
Standalone mode deployment
```
for consistency

  --set accessKey=admin,secretKey=admin123 \
  --set mode=standalone \
  --set service.type=NodePort \
  --set persistence.enabled=true \
  --set persistence.size=2Gi \
  --set persistence.storageClass=hwameistor-storage-lvm-hdd \
  minio/minio

[root@k8s-10-6-163-52 minio-test1]# kubectl get all -nminio-2
NAME                           READY   STATUS    RESTARTS   AGE
pod/minio-2-785f5c9985-7f5pf   1/1     Running   0          97m

NAME              TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)          AGE
service/minio-2   NodePort   10.104.40.2   <none>        9000:32000/TCP   97m

NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/minio-2   1/1     1            1           97m

NAME                                 DESIRED   CURRENT   READY   AGE
replicaset.apps/minio-2-785f5c9985   1         1         1       97m

View PVCs on HwameiStor

[root@k8s-10-6-163-52 minio-test1]# kubectl get pvc -nminio-2
NAME      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                 AGE
minio-2   Bound    pvc-3d4c1846-fc64-4af0-8104-64fc66c1c1bb   2Gi        RWO            hwameistor-storage-lvm-hdd   103m

> 3.2 Distributed mode deployment

Reviewer: ```suggestion
### Distributed mode deployment
```
